### PR TITLE
Revert "JP-2940 Return non-zero status from the set_telescope_pointin…

### PR DESCRIPTION
**Pull request by stscieisenhamer**
…g command-line when an error occurs (#7056)"

This reverts commit 6f6cef004aa16621bfdeef1a309eec5fb8cac1a1.

This reverts PR#7056 which caused unexpected failures in integration testing.

**Checklist for maintainers**
- [x] added entry in `CHANGES.rst` within the relevant release section
- [ ] updated or added relevant tests
- [ ] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [ ] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

### CODE REVIEW

**Comment by hbushouse** [comment]
Does this mean that a JP ticket now needs to be reset to Open or In Progress?

File: CHANGES.rst [hunk 1]
```diff
@@ -130,8 +133,6 @@ set_telescope_pointing
 
 - Migrate set_telescope_pointing to pysiaf-based exclusively [#6993]
 
-- Return non-zero status from the set_telescope_pointing command-line when an error occurs [#7056]
```
Comment (zacharyburnett) [inline CHANGES.rst:133]: I think this should remain, in case anyone tries to install `1.8.1` and has this issue

**Review by zacharyburnett — APPROVED** [review]
LGTM, aside from the changelog comment

**Comment by hbushouse** [comment]
Holding judgement till I see a correct change log! ;-)

**Comment by stscieisenhamer** [comment]
usual crds failures. pushing the changelog change...

**Review by hbushouse — APPROVED** [review]
LGTM

**Pull request merged**